core/citrus-base/src/main/java/org/citrusframework/actions/ReceiveMessageAction.java

```diff
@@ -336,11 +366,14 @@ private void assumeMessageType(Message message) {
 
                 setMessageType(MessageType.YAML);
             }
+        } else {
+            setMessageType(MessageType.PLAINTEXT);
```

Comment:
Since this is not really abvious, it might be worth a comment. I think it is done as such because there is always a plain message validator by default?

Reply:
good one.